package-overrides/github/gaearon/react-transform-catch-errors@1.0.0.json

```diff
@@ -0,0 +1,5 @@
+{
+  "registry": "npm",
+  "main": "src/index",
+  "format": "es6"
```

Comment:
Was there a reason you want to use the ES6 sources over the CommonJS transpiled sources?